web/src/components/apps/AppDetailPage.jsx

```diff
@@ -464,28 +473,32 @@ class AppDetailPage extends Component {
             }
           </div>
         </SidebarLayout>
-        {displayDownloadCommandModal &&
+        {displayRequiredKotsUpdateModal &&
           <Modal
-            isOpen={displayDownloadCommandModal}
-            onRequestClose={this.toggleDisplayDownloadModal}
+            isOpen={displayRequiredKotsUpdateModal}
+            onRequestClose={() => this.toggleDisplayRequiredKotsUpdateModal(null)}
             shouldReturnFocusAfterClose={false}
-            contentLabel="Download cluster command modal"
+            contentLabel="Required KOTS Update modal"
             ariaHideApp={false}
-            className="DisplayDownloadCommandModal--wrapper Modal"
+            className="DisplayRequiredKotsUpdateModal--wrapper Modal"
           >
             <div className="Modal-body">
-              <h2 className="u-fontSize--largest u-textColor--primary u-fontWeight--bold u-lineHeight--normal">Download assets</h2>
-              <p className="u-fontSize--normal u-textColor--bodyCopy u-lineHeight--normal u-marginBottom--20">Run this command in your cluster to download the assets.</p>
-              <CodeSnippet
-                language="bash"
-                canCopy={true}
-                onCopyText={<span className="u-textColor--success">Command has been copied to your clipboard</span>}
-              >
-                kubectl krew install kots
-                {`kubectl kots download --namespace ${this.props.appNameSpace} --slug ${this.props.match.params.slug}`}
-              </CodeSnippet>
+              <h2 className="u-fontSize--largest u-textColor--primary u-fontWeight--bold u-lineHeight--normal">You must update KOTS to deploy this version</h2>
+              <p className="u-fontSize--normal u-textColor--bodyCopy u-lineHeight--normal u-marginBottom--20">This version of {app?.name} requires a version of KOTS that is different from what you currently have installed. Follow the steps below to upgrade KOTS to the required version.</p>
+              {app?.isAirgap ?
+                <p className="u-fontSize--normal u-textColor--bodyCopy u-lineHeight--normal u-marginBottom--20">To get the version of KOTS specified in this release ({requiredKotsUpdateObj?.incompatibleKotsVersion}) you can visit the download portal or talk to your vendor. Once you have upgraded to the specified version you will be abel to deploy this version.</p>
```

Comment:
Typo: abel -> able
```suggestion
                <p className="u-fontSize--normal u-textColor--bodyCopy u-lineHeight--normal u-marginBottom--20">To get the version of KOTS specified in this release ({requiredKotsUpdateObj?.incompatibleKotsVersion}) you can visit the download portal or talk to your vendor. Once you have upgraded to the specified version you will be able to deploy this version.</p>
```

Reply:
message is different and coming from the API now